src/vs/workbench/contrib/notebook/browser/notebookBrowser.ts

```diff
@@ -49,6 +49,18 @@ export const JUPYTER_EXTENSION_ID = 'ms-toolsai.jupyter';
 export const KERNEL_EXTENSIONS = new Map<string, string>([
 	[IPYNB_VIEW_TYPE, JUPYTER_EXTENSION_ID],
 ]);
+// @TODO lramos15, place this in a similar spot to our normal recommendations.
+export const KERNEL_RECOMMENDATIONS = new Map<string, Map<string, INotebookExtensionRecommendation>>();
+KERNEL_RECOMMENDATIONS.set(IPYNB_VIEW_TYPE, new Map<string, INotebookExtensionRecommendation>());
```

Comment:
@sandy081 Could we somehow do something like this using a similar remote location to extension recommendations so it's not tied in code. I want to request an extension given a language and notebook type

Reply:
Not sure if I understand. Can you please give me more context.

Reply:
I see what you mean. I would have this list in product.json and not hardcode in the code.

Reply:
Right that was my idea I was wondering do you normally create a URL of a json list that we then put the URL in the product.json and fetch that URL? I was curious how this all worked and if there was a recommendation service or something that I could place it in